syncer/internal/controller/jobset.go

```diff
@@ -0,0 +1,179 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	v1 "github.com/llmariner/job-manager/api/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/utils/ptr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	jobset "sigs.k8s.io/jobset/api/jobset/v1alpha2"
+)
+
+const (
+	jobSetControllerName     = "jobset-controller"
+	fullJobSetControllerName = domain + "/" + jobSetControllerName
+)
+
+var jobSetGVR = schema.GroupVersionResource{
+	Group:    "jobset.x-k8s.io",
+	Version:  "v1alpha2",
+	Resource: "jobsets",
+}
+
+// JobSetController reconciles a Job object
+type JobSetController struct {
+	syncController
+	recorder record.EventRecorder
+}
+
+// SetupWithManager sets up the controller with the Manager.
+func (c *JobSetController) SetupWithManager(mgr ctrl.Manager, ssClient v1.SyncerServiceClient) error {
+	c.recorder = mgr.GetEventRecorderFor(fullJobSetControllerName)
+	c.k8sClient = mgr.GetClient()
+	c.ssClient = ssClient
+	c.controllerName = fullJobSetControllerName
+	return ctrl.NewControllerManagedBy(mgr).
+		Named(jobSetControllerName).
+		For(&jobset.JobSet{}).
+		Complete(c)
+}
+
+// Reconcile reconciles a local Job object and deploys it to the worker cluster.
+func (c *JobSetController) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	log := ctrl.LoggerFrom(ctx)
+
+	var jobSet jobset.JobSet
+	if err := c.k8sClient.Get(ctx, req.NamespacedName, &jobSet); err != nil {
+		err = client.IgnoreNotFound(err)
+		if err != nil {
+			log.Error(err, "Failed to get jobSet")
+		}
+		return ctrl.Result{}, err
+	}
+
+	if mgr := ptr.Deref(jobSet.Spec.ManagedBy, ""); mgr != c.controllerName {
+		log.V(4).Info("Skip jobSet set", "managedBy", mgr)
```

Comment:
```suggestion
		log.V(4).Info("Skip jobSet", "managedBy", mgr)
```

?

Or `Skip src set` or `Skip src` to be consistent with `job.go`?